packages/twenty-server/src/engine/metadata-modules/agent/agent-chat.service.ts

```diff
@@ -115,13 +118,13 @@ export class AgentChatService {
     return this.messageRepository.find({
       where: { threadId },
       order: { createdAt: 'ASC' },
-      relations: ['files'],
+      relations: ['parts', 'files'],
     });
   }
 
   private async generateTitleIfNeeded(
     threadId: string,
-    messageContent: string | null,
+    messageContent?: string | null,
```

Comment:
It doesn't really make sens to call generateTitle if message is empty no? Should we keep original typing and only call it when we have a defined message?